bionic/aip/main.py

```diff
@@ -32,6 +32,10 @@ def run():
 
 
 def _set_up_logging(job_id, project_id):
+    # We don't want to set up Stackdriver logging for local tests.
+    if os.environ.get("PYTEST_CURRENT_TEST", False):
```

Comment:
Having our real code check whether we're in a test seems hacky to me. Could we just have an explicit env variable or command line argument that configures whether to use stackdriver logging?